src/pages/docs/object-pick.mdx

```diff
@@ -20,7 +20,7 @@ const fish = {
   barckish: false
 }
 
-pick(fish, ['name', 'source']) // => { name, source }
+pick(fish, ['name', 'source']) // => { 'Bass', 'lake' }
```

Comment:
Hey @bontaramsonta thanks for the PR! This comment was actually intentional. I can see how its a bit misleading though because I was trying to keep it short. How about we just put the whole value:

```suggestion
pick(fish, ['name', 'source']) // => { name: 'Bass', source: 'lake' }
```